apps/web/src/services/dashboards/widgets/asset-widgets/trend-of-pass-and-fail-findings/TrendOfPassAndFailFindingsWidget.vue

```diff
@@ -161,16 +245,40 @@ const drawChart = (chartData: XYChartData[]) => {
             dateFormat: DATE_FORMAT,
             dateFields: [DATE_FIELD_NAME],
         });
-        // const tooltip = chartHelper.createTooltip();
-        // chartHelper.setXYSharedTooltipText(chart, tooltip, state.currency, props.currencyRates);
-        // series.set('tooltip', tooltip);
+        const tooltip = chartHelper.createTooltip();
+        tooltip.label.adapters.add('text', (text, target) => {
+            let totalValue = 0;
+            const seriesList: any[] = []; // { color: string, name: string, value: number }[]
+            chart.series.each((s) => {
+                const fieldName = s.get('valueYField') || s.get('valueXField') || '';
+                const value = target.dataItem?.dataContext?.[fieldName];
+                totalValue += value;
+                seriesList.push({
+                    color: s.get('stroke')?.toString() ?? '',
+                    name: s.get('name') ?? '',
+                    value: value ?? 0,
+                });
+            });
+            let _text = `Total: [bold; fontSize: 14px]${totalValue}[/]`;
+            seriesList.forEach((s) => {
+                const rate = Math.round((s.value / totalValue) * 100);
+                _text += `\n[${s.color}; fontSize: 10px]●[/] [fontSize: 14px;}]${s.name}:[/] [bold; fontSize: 14px]${s.value}[/] (${rate}%)`;
+            });
+            return _text;
+        });
+        series.set('tooltip', tooltip);
```

Comment:
description: Tooltip in this widget is little different from other widgets....🥲 So I custom here